src/main/java/pro/cloudnode/smp/nations/util/Nation.java

```diff
@@ -0,0 +1,130 @@
+package pro.cloudnode.smp.nations.util;
+
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+
+public class Nation {
```

Comment:
should also have a description I think

Reply:
could be displayed in list (maybe hover) and nation info

Reply:
oh you mean of the nation itself, i thought of like a javadoc class